samples/openapi3/client/petstore/go-experimental/go-petstore/api/openapi.yaml

```
@@ -1303,6 +1303,7 @@ components:
         id: 0
         email: email
         username: username
+        stuff: '{}'
```

IMHO it should be a JSON object not string '{}'.
```suggestion
        stuff: {}
```

---

You are correct, but this is generated code. There must be another bug in the existing code that causes this issue. Let me check.

---

This is generated from https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/go-experimental/openapi.mustache. There must be a problem with the content of the **openapi-yaml** tag.

---

Never mind, the generated code is correct and must be within quotes. This is because this line is part of the generated example value.
   
https://github.com/OpenAPITools/openapi-generator/blob/40470a619d74022e850a711e7e4643ab1fced06a/samples/openapi3/client/petstore/go-experimental/go-petstore/api/openapi.yaml#L1297

---

I am going to mark this issue resolved. Reopen if you have more questions.